resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -417,6 +417,17 @@ private[spark] object Config extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val KUBERNETES_EXECUTOR_MISSING_POD_DETECT_DELTA =
+    ConfigBuilder("spark.kubernetes.executor.missingPodDetectDelta")
+      .doc("When a registered executor's POD is missing from the Kubernetes API server's polled " +
+        "list of PODs then this delta time is taken as the accepted time difference between the " +
+        "registration time and the time of the polling. After this time the POD is considered " +
+        "missing from the cluster and the executor will be removed.")
+      .version("3.1.1")
```

Comment:
It seems that we need to change this to 3.0.2 in this case.

Reply:
Let's do this later in the follow-up because we need to update to `master/branch-3.1/branch-3.0` together.

Reply:
Ok. 

We have some other configs suffering from the safe problem, here for example:
https://github.com/apache/spark/blob/3afa99ae6393e9a3be88c24aa764e793839eec02/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala#L412-L416